lib/presentation/injector/injector.dart

```diff
@@ -46,7 +50,25 @@ class _Injector extends Injector {
     final KiwiContainer container = KiwiContainer();
     final SharedPreferences preferences = await SharedPreferences.getInstance();
     container
-        .registerSingleton<AppSettings>((c) => AppSettingsImpl(preferences));
+      ..registerSingleton<AppSettings>((c) => AppSettingsImpl(preferences))
+      ..registerSingleton<FirebaseService>((c) {
+        final service = FirebaseService();
+        print('initialising firebase service');
+        service.initialise();
```

Comment:
initialising twice?

Reply:
Oops I have recheck this I think i missed it